cmd/admin/setup/config.go

```diff
@@ -44,8 +46,12 @@ func MustNewSteps(v *viper.Viper) *Steps {
 
 	steps := new(Steps)
 	err = v.Unmarshal(steps,
-		viper.DecodeHook(hook.Base64ToBytesHookFunc()),
-		viper.DecodeHook(hook.TagToLanguageHookFunc()),
+		viper.DecodeHook(mapstructure.ComposeDecodeHookFunc(
+			hook.Base64ToBytesHookFunc(),
+			hook.TagToLanguageHookFunc(),
+			mapstructure.StringToTimeDurationHookFunc(),
+			mapstructure.StringToSliceHookFunc(","),
```

Comment:
these two are enabled by default

Reply:
and are overwritten as soon as you provide custom hooks:

> // DecodeHook returns a DecoderConfigOption which overrides the default
// DecoderConfig.DecodeHook value, the default is:
//
//  mapstructure.ComposeDecodeHookFunc(
//		mapstructure.StringToTimeDurationHookFunc(),
//		mapstructure.StringToSliceHookFunc(","),
//	)